canonical-web-and-design/vanilla-framework
# Vanilla 3.0 feature branch

Vanilla 3.0 feature branch.

Individual updates have been reviewed individually. This branch needs just a final QA before merging.

## QA

- Check demo: https://vanilla-framework-4070.demos.haus
- Make sure it works as expected
- Check if Percy doesn't show unexpected errors
